Makefile

```diff
@@ -54,8 +54,19 @@ install-arch: install install-mkinitcpio
 	install -dD $(DESTDIR)$(PREFIX)/share/libalpm/hooks
 	install -m0644 -t $(DESTDIR)$(PREFIX)/share/libalpm/hooks libalpm/hooks/95-m1n1-install.hook
 
+install-initramfs: install
```

Comment:
Can we use `initramfstools` here instead of `initramfs`? Otherwise it's confusing to people...